apps/nextjs-website/src/components/organisms/ChatbotLayout/ChatbotLayout.tsx

```diff
@@ -44,6 +45,10 @@ const ChatbotLayout = ({
   const [anchorEl, setAnchorEl] = React.useState<HTMLElement | undefined>(
     undefined
   );
+  const [formVisible, setFormVisible] = React.useState(false);
+  const [contextRelevancy, setContextRelevancy] = useState(0);
+  const [responseRelevancy, setResponseRelevancy] = useState(0);
```

Comment:
```suggestion
  const [contextRelevancy, setContextRelevancy] = useState(0);
  const [responseRelevancy, setResponseRelevancy] = useState(0);
```
I think you should set `null` as default value